hello-world/clients.js

```diff
@@ -0,0 +1,75 @@
+var express = require('express');
+var Client = require('node-rest-client').Client;
+var client = new Client();
+var router = express.Router();
+var rp = require('request-promise');
```

Comment:
@psinha20 Is this package 'request-promise' necessary here? If not, Please remove the pkg.